### DISCUSSION TITLE
pikpak无法加载任何视频和图像

### DISCUSSION THREAD

**Discussion by hibiki222** [AlistGo/alist#6514] [General]
pikpak无法加载任何视频和图像

**Comment by outsider1997** [comment]
确实，新版本可以挂载了，但是播放不了

**Comment by clbin001** [comment]
确实如此，我的服务器上也是这个问题，可以看到目录和文件列表，但是如果点击任何图片或视频文件，都无法播放

**Comment by tota666** [comment]
彻底完蛋了，pikpak现在是服务端加密，很难破解

**Comment by Muione** [comment]
现在好像限制了，请求下载链接的ip和下载的ip需要一样才可以，不然会返回403

**Comment by li-peifeng** [comment]
现在要开web代理，还要求IP相同

**Reply by unknowissue** [reply]
您好
这个开web代理 是把 WebDAV 策略 开成 使用代理地址的意思吗？IP相同应该如何实现呀？
谢谢

**Reply by li-peifeng** [reply]
web 策略上面有个web代理打开， 策略选本地代理

**Reply by clbin001** [reply]
非常感谢。根据你的指点，我挂载的pikpak又能正常播放啦。

**Reply by li-peifeng** [reply]
现在只能播放下载自己的pikpak了，分享的只能播放一半，下载也只能下载一半就错误，只有转存后才能正常播放下载
